### ISSUE TITLE
Deprecate Repository

### ISSUE THREAD

**Issue by djraval** [AdnanHodzic/google-podcasts-desktop#1]
Now it automatically redirects to the podcasts webapp.

**Comment by AdnanHodzic** [comment]
Right, just as described on: 

- https://github.com/AdnanHodzic/google-podcasts-desktop
- https://foolcontrol.org/?p=3095

So could you please elaborate what is this bug report exactly about?

**Comment by djraval** [comment]
Sorry, for the ambiguity of the message.
What I meant was that now the original url(google.com/podcasts) from Podcasts app automatically redirects to (podcasts.google.com) tested on latest chrome build.
I was planning to port your GoLang code to JS but today while testing it I noticed this change. I believe this tool's purpose was to change the url from Android app to WebApp's format but that purpose is already served by google's redirection.

**Comment by AdnanHodzic** [comment]
I see, I just copied URL generated by Google Podcasts Android app to my notebook and indeed proper URL redirect was automatically made by Google's backend. 

I just hope Google rolled out these changes [7 days ago](https://camo.githubusercontent.com/51f7c4dd785ef42ca0db34c505bd4e86601b578f/68747470733a2f2f666f6f6c636f6e74726f6c2e6f72672f77702d636f6e74656e742f75706c6f6164732f323031392f30352f676f6f676c652d706f64636173742d6465736b746f702d73637265656e636173742d332e676966), before I made all this effort :) But who knows, maybe creating one such tool was a nudge they needed to make this functionality widely available. 

Regardless, thank you for reporting this. I won't delete the repository, but I'll add a notice and point to this issue in both [blog post](https://foolcontrol.org/?p=3095) and [repo ReadMe](https://github.com/AdnanHodzic/google-podcasts-desktop) file. As with latest changes they rolled out in meantime, truly make this tool redundant.
